pulp_container/app/migrations/0009_get_upload_path.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.16 on 2020-10-21 10:48
+
+import django.core.files.storage
+from django.db import migrations, models
+import pulp_container.app.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('container', '0008_include_exclude_tags'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='upload',
+            name='file',
+            field=models.FileField(max_length=255, null=True, storage=django.core.files.storage.FileSystemStorage(location=pulp_container.app.models.Upload._get_upload_path), upload_to=pulp_container.app.models.generate_filename),
+        ),
+    ]
```

Comment:
Fun fact: I think this migration adheres to the yet to define zero-downtime policy. :+1: